<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushFront_PushButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>400</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Push Front</string>
    </property>
   </widget>
   <widget class="QPushButton" name="displayList_PushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Display Linked List</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="selectPortion_GroupBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>170</y>
      <width>121</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Select Portion</string>
    </property>
    <widget class="QLabel" name="start_Label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="endInputPortionList_LineEdit">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>51</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="startInputPortionList_LineEdit">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>51</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="end_Label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>End</string>
     </property>
    </widget>
    <widget class="QPushButton" name="displayPortion_PushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>101</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Display Portion</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="chooseCopy_PushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose Copy</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pushBackInput_LineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sort_PushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>221</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Sort</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="listOutput_TextEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>221</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="insertSorted_PushButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>430</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Insert Sorted</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="currentCopyNum_LCDNumber">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="makeCopy_PushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>221</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Make Copy</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="insert_GroupBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>320</y>
      <width>121</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Insert</string>
    </property>
    <widget class="QLabel" name="value_Label">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Value</string>
     </property>
    </widget>
    <widget class="QLabel" name="node_Label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Node</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="insertValueInput_LineEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>41</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="insertNodeInput_LineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>41</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="insert_PushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>101</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Insert</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="popFront_PushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>221</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Pop Front</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pushFrontInput_LineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="insertSortedInput_LineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBack_PushButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>370</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Push Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeDuplicates_PushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>221</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove Duplicates</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="nodeType_GroupBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>121</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Choose Node Type</string>
    </property>
    <widget class="QRadioButton" name="complex_RadioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>100</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Complex</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="string_RadioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>100</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>String</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="double_RadioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>100</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Double</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rational_RadioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>100</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Rational</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="int_RadioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>100</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Int</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="date_RadioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>100</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Date</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>393</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOsbaldo_Gonzalez_s_Linked_List_Display">
    <property name="title">
     <string>Osbaldo Gonzalez's Linked List Display</string>
    </property>
   </widget>
   <addaction name="menuOsbaldo_Gonzalez_s_Linked_List_Display"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
